django/contrib/auth/management/commands/createsuperuser.py

```diff
@@ -137,9 +137,11 @@ def handle(self, *args, **options):
                         if not field.many_to_many:
                             fake_user_data[field_name] = input_value
 
-                        # Wrap any foreign keys in fake model instances
-                        if field.many_to_one:
-                            fake_user_data[field_name] = field.remote_field.model(input_value)
+                    # Wrap any foreign keys in fake model instances
+                    if field.many_to_one:
+                        user_data[field_name] = fake_user_data[field_name] = field.remote_field.model(
+                            user_data[field_name]
+                        )
```

Comment:
There is no reason why the value should be wrapped only when it was passed by input. The user can have provided some values using command line. I spotted this bug while working on [#33151](https://code.djangoproject.com/ticket/33151). Please tell me if it's ok to fix it in the same PR.

`test_fields_with_fk_interactive_and_value_provided` tests this case.